components/LogoBackground.js

```diff
@@ -0,0 +1,29 @@
+const LogoBackground = ({ width, top = 0, left, right, bottom }) => (
```

Comment:
Since we assume that the logo’s height is equal to the width (in line 2), I suggest changing the `width` property to “size”.